python/strings_udf/strings_udf/_lib/cudf_jit_udf.pyx

```diff
@@ -22,3 +25,18 @@ def to_string_view_array(Column strings_col):
 
     device_buffer = DeviceBuffer.c_from_unique_ptr(move(c_buffer))
     return Buffer(device_buffer)
+
+
+def from_udf_string_array(DeviceBuffer d_buffer):
```

Comment:
Can we rename to?

```suggestion
def column_from_udf_string_array(DeviceBuffer d_buffer):
```

Reply:
did this and also renamed `to_string_view_array` -> `column_to_string_view_array` in https://github.com/rapidsai/cudf/pull/11933/commits/e864deae2e7933dc796cbf910a6caae1da11b028